test/apps/test_fuzzy_match.py

```diff
@@ -16,72 +16,75 @@
 class Environment(object):
     '''Handles environment variables setup and cleanup'''
     def __init__(self, env_vars):
-        logging.info('Using env vars: %s' % env_vars)        
+        logging.info('Using env vars: %s' % env_vars)
         self.env_vars = env_vars;
-        
+
     def __enter__(self):
         self.cleanup()
         for var_name in self.env_vars:
             os.environ[var_name] = 'value'
         return self
-    
+
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.cleanup()
-        
+
     def cleanup(self):
         ucx_vars = [var for var in os.environ.keys() if var.startswith('UCX_')]
         for var in ucx_vars:
             del os.environ[var]
 
-    
+
 class TestRunner:
     '''Main test runner'''
     def __init__(self, ucx_info, verbose):
         self.ucx_info = ucx_info
         if verbose:
             logging.basicConfig(level=logging.DEBUG)
-       
+
     def run(self, test_case):
         with Environment(test_case.keys()):
             matches = self.get_fuzzy_matches()
 
             if matches != test_case:
                 raise Exception('Wrong fuzzy list: got: %s, expected: %s' % (matches, test_case))
-        
+
             logging.info('found all expected matches: %s' % test_case)
-        
+
     def exec_ucx_info(self):
         cmd = self.ucx_info + ' -u m -w'
         logging.info('running cmd: %s' % cmd)
-    
+
         status, output = commands.getstatusoutput(cmd)
         if status != 0:
             raise Exception('Received unexpected exit code from ucx_info: ' + str(status))
-            
+
         logging.info(output)
         return output
-    
+
     def get_fuzzy_matches(self):
         output = self.exec_ucx_info()
-        warn_msg = output.splitlines()[0]
-        
+        output_lines = output.splitlines()
+        warn_msg = output_lines[0]
+
         # This text is printed from 'parser.c' file (updates should be synced properly).
         warn_match = re.match('.*unused environment variables?: (.*)', warn_msg)
         if not warn_match:
+            print("First 5 lines of output:")
```

Comment:
why not all lines?

Reply:
it is too much output, and first several lines should be enough to check why there is no warning on the first line